raft/raft.go

```diff
@@ -110,7 +110,7 @@ type Config struct {
 
 	// Logger is the logger used for raft log. For multinode which can host
 	// multiple raft group, each raft group can have its own logger
-	Logger Logger
+	Logger *DefaultLogger
```

Comment:
can we just remove it and move `/kv/util/log` to `log`?

Reply:
In some tests, the different loggers will be used for logging. And they have different prefixes and writers. So we should create a new type to wrap the default logger then they could be replaced in runtime.